.github/workflows/clang-tidy.yml

```diff
@@ -35,4 +35,4 @@ jobs:
     - name: run
       shell: bash
       working-directory: ${{ runner.workspace }}/_build
-      run: run-clang-tidy -checks=*,-clang-analyzer-deadcode*,-clang-analyzer-optin*
+      run: run-clang-tidy -checks=*,-clang-analyzer-deadcode*,-clang-analyzer-optin*,-fuchsia-*,-llvmlibc-*,-modernize-use-trailing-return-type
```

Comment:
You might want to place this in a `.clang-tidy` file in the root source directory

Reply:
yeah i'm also going to reduce the number of checks we're running.